File: python/plot.c
{
        int N;
TFile f("../run/root_jpsiantisigmaminussigmaplus/jpsiantisigmaminussigmaplus_gen_mc.root", "read");
gStyle->SetOptStat(0);
TTree* tree = (TTree*) f.Get("tree");
TH1F *h = new TH1F("", "", 1000, -20 ,20);
//TH1F *h1 = new TH1F("", "", 1000, 0 ,3);
//TH1F *h2 = new TH1F("", "", 1000, 0 ,3);
double x,y,z;
tree->SetBranchAddress("vr0", &x);
//tree->SetBranchAddress("br_y", &y);
//tree->SetBranchAddress("br_z", &z);
N=tree->GetEntries();
for (int i=0;i<N;i++){
        tree->GetEntry(i);
        h->Fill(x);
       // h1->Fill(z);
       // h2->Fill(x);
}
h->SetLineColor(kRed);
//h2->SetLineColor(kYellow);
h->Print();
h->GetXaxis()->SetTitle("vr0");
h->GetYaxis()->SetTitle("entries");
h->Draw();
h->SaveAs("plot2.pdf");
//h1->Draw("same");
//h2->Draw("same");

}

File: python/plot.py
#!/usr/bin/env python
"""
Generate plots
"""

__author__ = "AMIT PATHAK <amit@ihep.ac.cn>"
__copyright__ = "Copyright (c) AMIT PATHAK"
__created__ = "[2019-07-22 Tue 15:00]"

import ROOT
from ROOT import gStyle, TCanvas, TString, TChain, TStyle
from ROOT import TH1D, TFile, TLegend

f = ROOT.TFile.Open('../run/root_jpsiantisigmaminussigmaplus/jpsiantisigmaminussigmaplus_gen_mc.root', 'read')
canvas = ROOT.TCanvas('canvas','',500,500)
tr1 = f.Get('tree')
tr1.Draw('vr0')
canvas.SaveAs('../run/plots/Plot.pdf')
f.Print()
f.Close()
